Mesh4all/m4a-firmware
# wifi-subsys: fix wrong macro literal to string

### Contribution description
This fix a bug related to converting literals to string
### Testing procedure
Type and execute the following in the /test folder:
```sh
idf.py build -D "TEST_COMPONENTS=storage" flash monitor
```
Select the test case `#1` , the expected result should be:
```sh
Running Literal to String...
I (346) TEST: stringlify: TEST ?= expected: TEST
...m4a-firmware/wifi-subsys/components/storage/test/test_nvs.c:20:Literal to String:PASS
```
### Issues/PRs references
Fix #127
